backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/storage/lsm/LiveMigrateDiskCommand.java

```diff
@@ -290,7 +293,7 @@ public boolean performNextOperation(int completedChildCount) {
             // at the end of CreateSnapshotForVm command called from the executeCommand() method.
             // Here the lock is acquired again and will be released when this command (LiveMigrateDisk)
             // finishes.
-            if (!lockManager.acquireLock(getLock()).isAcquired()) {
+            if (!lockManager.acquireLock(removeSnapshotLock).isAcquired()) {
```

Comment:
I also suggest to handle this issue:
https://bugzilla.redhat.com/show_bug.cgi?id=2110186#c3

Reply:
Don't know if it is neccessary in this particular case

Reply:
@ahadas wdyt?

Reply:
after this change, it would be much more likely that it is the VM lock that prevent us from taking the engine lock so I see no reason to couple it with this PR, but yeah, it would be better to log the message that is returned by the lock manager - we can do that in a separate PR